packages/website/components/LiveEditor/StaticSource.tsx

```diff
@@ -11,6 +11,7 @@ const styles = {
   root: css({
     padding: tokens.spacingM,
     borderRadius: tokens.borderRadiusMedium,
+    marginBottom: tokens.spacingL,
```

Comment:
```suggestion
    marginBottom: tokens.spacingM,
```

looking the picture you showed,
I think it should have the same marginBottom as a paragraph